jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/store/GraphViewSwitchable.java

```diff
@@ -79,16 +83,34 @@ public TransactionHandler getTransactionHandler() {
     /** Return the {@code DatasetGraphSwitchable} we are viewing. */
     @Override
     public DatasetGraphSwitchable getDataset() {
-        return getx() ;
+        return getx();
     }
     
     /** Return the {@code Graph} from the underlying switchable.
      *  Do not hold onto this reference across switches. 
      */
     public Graph getGraph() {
-        return getx().getGraph(getGraphName()) ;
+        return getx().getGraph(getGraphName());
     }
 
+    // Super uses find. Override to call GraphTDB.size()
+    @Override
+    protected int graphBaseSize() {
+        if ( isDefaultGraph() )
+            return getDSG().getDefaultGraphTDB().size();
+        return getDSG().getGraphTDB(getGraphName()).size();
+    }
+
+    private DatasetGraphTDB getDSG() {
+        return ((DatasetGraphTDB)(getx().get()));    
+    }
+    
+    private static Function<Tuple<NodeId>, Tuple<NodeId>> project4TupleTo3Tuple = item -> {
```

Comment:
This seems kind of out of place...

Reply:
Don't understand.  It is used in the method above and pulled out for clarity. It makes the handling of union graph clearer IMO with a named function at that point, not inline code.

Reply:
I'm totally confused. The method above reads:
```
private DatasetGraphTDB getDSG() {
        return ((DatasetGraphTDB)(getx().get()));    
}
```
Where is `project4TupleTo3Tuple` used there?

Reply:
OK - got it - it's a stray from GraphTDB. This one can be cleaned out.